### ISSUE TITLE
Create a new badge for teams that meet the challenges posed by other teams

### ISSUE THREAD

**Issue by darozak** [darozak/BioTreks#27]
The idea is to create a badge that encourages students to build on the work presented by other.   The award will be offered to papers which accept another team's laboratory challenge and present their results in a research paper on the topic.

**Comment by darozak** [comment]
The plan is to complete this by **22 January** so that it is available for a general public announcement.

**Comment by darozak** [comment]
I just created two new badges to recognize **challengers* and **champions** (https://github.com/darozak/BioTreks/commit/fad6c59ea64980bee32437cfc374a605d906f473).  My nest step will be to fill in the details for theses badges so I can present them to teams and judges.

**Comment by darozak** [comment]
I decided to stick with existing badges but redefine Collaboration as articulating and pursuing community goals.  This will allow for continuity among the badges that are being awarded from one year to the next.  I've revised the badge SOP to capture this approach (https://github.com/darozak/BioTreks/commit/5aa8fe0b4c61a3ff5ff2049441b8f498b6ec7cb2). Since this strategy uses existing badges, I won't need to commission the design of new badges.  I can close out this issue.
